src/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import App from './App';
```

Comment:
[Part 3 – How to implement CSS in Reactjs app](https://ibaslogic.com/css-in-reactjs-app/). is missing, please complete this task to add CSS to the project